src/main/java/gyro/google/iam/ServiceAccountKeyFinder.java

```diff
@@ -0,0 +1,52 @@
+package gyro.google.iam;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.google.api.services.iam.v1.Iam;
+import com.google.api.services.iam.v1.model.ServiceAccountKey;
+import gyro.core.Type;
+import gyro.google.GoogleFinder;
+
+/**
+ * Query service account keys.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    service-account-key: $(external-query google::service-account-key {service-account-id: "projects/aerobic-lock-236714/serviceAccounts/example-service-account@aerobic-lock-236714.iam.gserviceaccount.com"})
```

Comment:
Do not include project sensitive info
```suggestion
 *    service-account-key: $(external-query google::service-account-key {service-account-id: ""})
```